src/containers/Trainee/AttendanceDetails.tsx

```diff
@@ -10,18 +10,18 @@ function AttendanceDetails() {
   const { t } = useTranslation();
   const navigate = useNavigate();
   return (
-    <div>
+    <div className='bg-neutral-100 dark:bg-dark-frame-bg'>
```

Comment:
Can we fix this spacing

![image](https://user-images.githubusercontent.com/28519395/183630602-fb79986b-f717-4fe8-b3fc-1aec150529a9.png)

There is a large gap here

Reply:
Solved.